state.json

```diff
@@ -0,0 +1,2 @@
+{"U1RBVEU=":"AgAAAAMAAABib2IFAAAAd29ybGQNAAAAcmVwcm8udGVzdG5ldAUAAABoZWxsbw"}
```

Comment:
What is this and why it is in the root?

Reply:
sorry, added by mistake when resolve conflicts